sdk/src/main/scala/io/horizen/forge/AbstractForgeMessageBuilder.scala

```diff
@@ -154,6 +154,10 @@ abstract class AbstractForgeMessageBuilder[
     val currentTipBlockEpochAndSlot: ConsensusEpochAndSlot = TimeToEpochUtils.timestampToEpochAndSlot(params.sidechainGenesisBlockTimestamp, currentTipBlockTimestamp)
     val nextBlockEpochAndSlot: ConsensusEpochAndSlot = ConsensusEpochAndSlot(nextEpochNumber, nextSlotNumber)
 
+    if ((nextEpochNumber - currentTipBlockEpochAndSlot.epochNumber) > 1) {
```

Comment:
I'm probably just being paranoid but if for any reason for 2 epochs no block is forged, the forging will be stopped forever. I know that it shouldn't be possible but what if there is an attack? Or what if at the beginning of a chain life for some reason we don't have blocks for 2 epochs? 
So, just to err on the safe side, what about creating a configuration param that can disable this check?

Reply:
The suggested approach will be analyzed with the team in a separate context, and in case a new issue will be open for working at it.

Reply:
ok for me